pkgs/test/nixpkgs-check-by-name/src/eval.rs

```diff
@@ -39,11 +39,10 @@ enum AttributeVariant {
 /// of the form `callPackage <package_file> { ... }`.
 /// See the `eval.nix` file for how this is achieved on the Nix side
 pub fn check_values(
-    version: Version,
     nixpkgs_path: &Path,
     package_names: Vec<String>,
-    eval_accessible_paths: Vec<&Path>,
-) -> validation::Result<()> {
+    eval_accessible_paths: &Vec<&Path>,
```

Comment:
One "rusty" nit is to have the various things accepting `&Vec<Something>` accept instead the slice `&[Something]`. That's both lower power and more general. `&Vec<T>` automatically [Derefs to `&[T]`](https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-Deref-for-Vec%3CT,+A%3E).

Reply:
It could even be `impl IntoIterator<Item = impl AsRef<Path>>` to be really general, since all you do with this value is iterate over it.

Reply:
`impl` feels a bit overkill. This doesn't expose any public interface, so it should be fine for it to not be polymorphic to make it easier to read.

The `&[&Path]` in this case sounds good to me!